### PR TITLE
Unit test for server/deprecations

### PR DESCRIPTION
**Pull request by TehomCD**
#### Brief overview of PR changes/additions
Yay Hacktoberfest! I'm trying to get more in the habit of writing unit tests, so I'll just be trying to write some for modules that need coverage.

#### Motivation for adding to Evennia
Provide test coverage for the deprecation warnings/errors module.

#### Other info (issues closed, discussion etc)
Part of #1458

### CODE REVIEW

File: evennia/server/tests.py [hunk 1]
```diff
@@ -46,3 +43,37 @@ def build_suite(self, test_labels, extra_tests=None, **kwargs):
         import evennia
         evennia._init()
         return super(EvenniaTestSuiteRunner, self).build_suite(test_labels, extra_tests=extra_tests, **kwargs)
+
+
+class MockSettings(object):
```
Comment (Griatch) [inline evennia/server/tests.py]: This works, but I've recently learned there are better ways to mock the django settings. Actually the Django TestCase class contains a context manager for this. So you can do in your test:

```python
class TestIdle(django.tests.TestCase):
    def test_idle(self):
        with self.modify_settings(IDLE_TIMOUT=1000):
            # stuff testing a different idle timout
       # here the original value is back again
```

Good stuff to remember for the future. :)

**Comment by Griatch** [comment]
@TehomCD Looks good, but unit tests should always be done against Master Branch. So I'll have to cherry pick these, meaning this will never look like it merges.

**Comment by Griatch** [comment]
@TehomCD I squashed your commits and cherry-picked them into master. Thanks for the contribution!

File: evennia/server/tests.py (hunk 1 above)
Comment (TehomCD) [reply evennia/server/tests.py]: Ooh, good to know, thanks!

**Pull request closed without merging**